src/convertToObject.js

```diff
@@ -5,8 +5,24 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = {};
+
+  const rules = sourceString
+    .split(/;(?![^(]*\))/)
+    .filter((rule) => rule.trim() !== '');
+
+  rules.forEach((rule) => {
+    const [property, ...valueParts] = rule.split(':');
```

Comment:
The destructuring assignment is correct, but ensure that the `property` is also trimmed before checking its existence to avoid issues with properties that may have leading or trailing whitespace.